bibtexparser/tests/test_bibdatabase.py

```diff
@@ -24,6 +24,15 @@ def test_entries_dict_prop(self):
         bib_db.entries = self.entries
         self.assertEqual(bib_db.entries_dict, bib_db.get_entry_dict())
 
+    def test_equality(self):
+        bib_db1 = BibDatabase()
+        bib_db2 = BibDatabase()
+        assert bib_db1 == bib_db2
+        bib_db1.entries = self.entries
+        assert bib_db1 != bib_db2
+        bib_db2.entries = self.entries
+        assert bib_db1 == bib_db2
```

Comment:
please also test the other attributes (other than `.entries`). Obviously, the implementation is correct - but adding a test for each field can help us avoid nasty regressions :-)

Reply:
done

Reply:
I've checked a bit the doc, but honestly, I couldn't find an example. Is this correct:
```
    comments = ['a', 'b']
    strings = OrderedDict(c="c", d="d")
    preambles = ['e', 'f']
```

Especially for strings, are the values strings or maybe `BibDataString`?

Reply:
Excuse my late reply. I'll need to dig into this a bit more. Unfortunately, I currently don't have much time and also must prioritize v2. I'll get back to this PR as soon as possible. Thanks for your understanding!